WordPress/Resources/bg.lproj/Localizable.strings

```diff
@@ -1503,6 +1503,9 @@ translators: Block name. %s: The localized block name */
 /* Title displayed for selection of custom app icons that have colorful backgrounds. */
 "Colorful backgrounds" = "Colorful backgrounds";
 
+/* translators: %d: column index. */
+"Column %d" = "Column %d";
```

Comment:
This is the particular string that we needed in the localizations 😉 as per https://github.com/wordpress-mobile/gutenberg-mobile/pull/3952